Microsoft.DotNet.DarcLib.Tests/RemoteTest.cs

```diff
@@ -0,0 +1,110 @@
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Xunit;
+
+namespace Microsoft.DotNet.DarcLib.Tests
+{
+    public class RemoteTest
+    {
+        Mock<IGitRepo> client = new Mock<IGitRepo>();
```

Comment:
There doesn't seem to be a good reason for these to be fields.  Just have them be local variables in the test itself (so that this class isn't dependent on how the testing framework deal with class instantiation.